### PR TITLE
#72 - Add code styling standards for contributors

### PR DESCRIPTION
**Pull request by BernardoSilva**
This PR is for issue #72

### CODE REVIEW

**Comment by tmilos** [comment]
Why don't we put this to be executed on CI rather then as commit hook?

**Comment by BernardoSilva** [comment]
To ensure that developers always commit good code.
This way if a developer make a change to code it will always commit clean formatted code, all consistent to whole project because the rules are the same to everyone. 
If we add this to CI it will check just for invalid syntax? 
Or event if we managed to make CI fix the code it will create strange commits just for code formatting? 
Or you know some better way of doing it @tmilos ?

**Comment by tmilos** [comment]
Didn't notice it's actually modifying files. Thought it's just syntax check. I'm used more to CI PSR check, but this might be fine too. 

Was a bit skeptical how it would work on windows, but after fixing line 26 to be 
```
        if [[ $i == *.php ]];
```
it works like a charm

So, please try if that works for you too, on Linux I guess, and we can merge it with that patch.

File: contrib/pre-commit [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+numberFilesChanged="${#filenames[@]}"
+
+if [[ $numberFilesChanged > 0 ]];
+then
+    echo "$numberFilesChanged files were changed, running php-cs-fixer"
+    # PHP CS Fixer.
+    for i in "${filenames[@]}"
+    do
+        if [[ $i =~ \.php$ ]];
```
Comment (tmilos) [inline contrib/pre-commit]: Does not work on windows 1.9.5.msysgit.1. Try if it works for you like
```
        if [[ $i == *.php ]];
```

**Comment by BernardoSilva** [comment]
@tmilos Its working fine on mac. :+1:

**Comment by tmilos** [comment]
@BernardoSilva  What are ``-p`` and ``--diff`` options? I don't find them in ``phpcbf -h``

**Comment by tmilos** [comment]
I made PR #85 as alt to this one to solve conflicts. I also changed ``composer.json`` to require stable version of php_codesniffer. Guess that's ok

**Comment by BernardoSilva** [comment]
Cool. :+1: 
Those options may be available only for `php-cs-fixer`, I adapted this script to work with php_codesniffer. 
Those may be incompatible options with this bundle.

**Pull request closed without merging**